style.css

```diff
@@ -284,3 +287,96 @@ p {
   font-weight: 500;
   padding-bottom: 1.5em;
 }
+
+/* styling the contact form */
+form input,
+form textarea {
+  display: block;
+  padding: 1em;
+  margin: 2em 1em;
+  width: 80%;
+  outline: none;
+  border: 1px solid #d0d9d4;
+}
+
+form input:focus,
+form textarea:focus {
+  border: 1px solid #60c095;
+}
+
+.contact-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+}
+
+#contact-section h2 {
+  text-align: center;
+  font-weight: bold;
+  padding: 2em 1em;
+}
+
+form {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  max-width: 300px;
+}
+
+#form button {
+  display: block;
+  margin: 0 auto;
+}
+
+.form-btn {
+  /* style the button */
+  border: 0;
+  padding: 12px;
+  width: fit-content;
+  width: -moz-fit-content;
+  height: 48px;
+
+  /* Theme 2 / Primary 2 */
+  background: #36b37f;
+  border-radius: 4px;
+  text-decoration: none;
+  color: #fff;
+  font-size: large;
+  cursor: pointer;
+}
+
+.form-btn:hover {
+  cursor: pointer;
+  background: #3ec48c;
+  box-shadow: 1px 8px 16px rgba(54, 179, 127, 0.24);
+}
+
+.form-btn:focus {
+  cursor: pointer;
+  background: #01a566;
+}
+
+#contact-section p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+
+  /* or 160% */
+  text-align: center;
+
+  /* Neutrals/N500 */
+  color: #42526e;
+  margin-top: 1em;
+  margin-bottom: 3em;
+}
+
+.end-shape {
+  margin-top: 5em;
+  margin-bottom: 1em;
+}
+
+#form img {
+  position: absolute;
+  right: 0;
```

Comment:
- [ ] Good job so far by working on the project requirements as expected, but there is some issues that still exists. Please, kindly implement the your project exactly as the Figma design. This is to make sure that you follow the Figma design guideline and replicate the design to look almost identical to the original design.
### your design:
![image](https://user-images.githubusercontent.com/71874425/159580726-51929537-7621-454e-919f-2b932f563e5f.png)
- Make sure that the text in textarea is in bold style.
- kindly, replace the image background on is exact place following the figma design.

Reply:
okey, I just forget to add the id ="form" to html file, u add it to fix stylelint error about { no-descending-specificity}

Reply:
Done! Thank you for the observation.